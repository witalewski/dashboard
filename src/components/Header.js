import React from 'react';
import styled from '@emotion/styled';
import { borderColor } from '../global/styleConstants';

const HeaderStyled = styled.header`
  width: 100%;

  padding: 24px 36px;

  background: white;
  border: 1px solid ${borderColor};

  .title {
    font-weight: normal;
    font-size: 24px;
  }
`;

export const Header = ({ title }) => (
  <HeaderStyled>
    <h1 className="title">{title}</h1>
  </HeaderStyled>
);
