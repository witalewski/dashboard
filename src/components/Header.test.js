import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import { Header } from './Header';

describe('Header', () => {
  it('matches snapshot', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<Header title="Dashboard" />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
