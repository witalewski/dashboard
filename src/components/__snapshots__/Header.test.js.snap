// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<ForwardRef(render)>
  <h1
    className="title"
  >
    Dashboard
  </h1>
</ForwardRef(render)>
`;
